packages/integration-react/src/WidgetWrapper.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+/**
+ * Parses container selector into element property. Currently
+ * only works with IDs and classnames.
+ *
+ * @param {string} [containerSelector='']
+ * @return {{ string: string }} Property name and value tuple.
+ */
+function selectorToAttribute(containerSelector = '') {
+  if (typeof containerSelector !== 'string' || containerSelector.length === 0) {
+    return {};
+  }
+
+  const lastPart = containerSelector.split(' ').pop();
+
+  return {
+    [lastPart[0] === '#' ? 'id' : 'className']: lastPart.substr(1),
```

Comment:
It fail for more specific selector like that `'.e_dU.atm-list-ul.d_gX.d_gZ'`.

Reply:
good point, I've refactored the function to be more robust. It still parses only IDs and classnames for now but I recon we can live with that for now.